intro.md

```diff
@@ -247,9 +279,9 @@ Note: example ECG sensors. First, all data is processed at the computer of the d
 <!--v-->
 ## Many ways to process the data
 
-- Many frameworks for stream processing
+- Many frameworks for (stream) processing
```

Comment:
why put parenthesis here? The topic is stream processing, after all.

Reply:
That's to deliberately start a discussion, which apparently works, thanks for testing 🤣 
Not all analyses are stream-based, some are batch-based or event-based. To ensure the trainees don't experience any potential limitations, I want to give space for a brief discussion in the direction of use-cases.